README.md

```diff
@@ -4,10 +4,64 @@
 [![Coverage Status](https://coveralls.io/repos/github/andela/knights-bn-backend/badge.svg?branch=develop)](https://coveralls.io/github/andela/knights-bn-backend?branch=develop)
 [![Reviewed by Hound](https://img.shields.io/badge/Reviewed_by-Hound-8E64B0.svg)](https://houndci.com)
 
-Barefoot Nomad - Making company travel and accomodation easy and convinient.
-=======
-
+# Barefoot Nomad
```

Comment:
Let us have `Barefoot Nomad` as the first statement in the README instead of badges.

Reply:
you mean the title and the description ? or just the title only!

Reply:
Have the title as `Barefoot Nomad` and remove the word badges.

Reply:
ok